tests/Makefile.am

```diff
@@ -95,3 +97,21 @@ test_scribe_LDADD = \
 test_scribe_LDFLAGS = \
 	$(WARN_LDFLAGS) \
 	$(NULL)
+
+test_unattended_config_SOURCES = test-unattended-config.c
+test_unattended_config_CFLAGS = \
+	$(INITIAL_SETUP_CFLAGS) \
+	$(IMAGE_INSTALLER_CFLAGS) \
+	-I $(top_srcdir)/ext/libglnx \
+	-I $(top_srcdir)/gnome-image-installer/util \
```

Comment:
Nitpick: Technically `-I` `CPPFLAGS` yadda yadda autotools losing the will to care about this kind of thing.

Reply:
I'll leave this one because maybe one day, a new build system.